common/build/eslint-plugin-fluid/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "@fluid-internal/eslint-plugin-fluid",
+	"version": "0.1.0",
+	"private": true,
```

Comment:
You'll need to remove this line before we can publish the package. But if you want to do that in a subsequent PR, that definitely works :)